packages/imperative/__tests__/config/cmd/profiles/profiles.handler.test.ts

```diff
@@ -0,0 +1,101 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+jest.mock("../../../../../utilities/src/ImperativeConfig");
+
+import { IConfig } from "../../../../../config";
+import { ImperativeConfig } from "../../../../../utilities";
+import ProfilesHandler from "../../../../src/config/cmd/profiles/profiles.handler";
+
+let dataObj: any;
+let formatObj: any;
+let errorText: string;
+let logText: string;
+
+// "Mocked" version of the handler parameters for a config list command
+const handlerParms: any = {
+    response: {
+        data: {
+            setObj: jest.fn((jsonObj) => {
+                dataObj = jsonObj;
+            })
+        },
+        format: {
+            output: jest.fn((formatArgs) => {
+                formatObj = formatArgs.output;
+            })
+        },
+        console: {
+            log: jest.fn((msgText) => {
+                logText = msgText;
+            }),
+            error: jest.fn((msgText) => {
+                errorText = msgText;
+            })
+        }
+    }
+};
+
+const configProps: IConfig = {
+    profiles: {
+        email: {
+            profiles: {
+                incoming: {
+                    type: "imap",
+                    properties: {
+                        host: "fakeHost",
+                        port: 143
+                    }
+                },
+                outgoing: {
+                    type: "smtp",
+                    properties: {
+                        host: "fakeHost",
+                        port: 25
+                    }
+                }
+            },
+            properties: {}
+        }
+    },
+    defaults: {},
+    plugins: [],
+    secure: []
+};
+
+describe("Configuration List command handler", () => {
```

Comment:
A minor detail, but this looks like the description copied from the list command handler. Should it say 'profile command handler'?